src/main/java/net/openhft/chronicle/threads/EventGroup.java

```diff
@@ -277,8 +277,16 @@ public void addTimingMonitor(final String description,
      */
     @Override
     protected void performStart() {
-        if (core != null)
+        if (core != null) {
             core.start();
+
+            while (!isAlive()) {
```

Comment:
we already do this, further down the function - see https://github.com/OpenHFT/Chronicle-Threads/blob/d6e59a039ca0e7347892bd74627b6e1ad8d80b13/src/main/java/net/openhft/chronicle/threads/EventGroup.java#L298

Why don't we just move that condition further up the function then we should not have to make any other changes?

Reply:
for example - https://github.com/OpenHFT/Chronicle-Threads/pull/183